src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/DetectPreviewFeatureAnalyzer.cs

```diff
@@ -82,13 +85,23 @@ public override void Initialize(AnalysisContext context)
                 ConcurrentDictionary<ISymbol, bool> requiresPreviewFeaturesSymbols = new();
                 ConcurrentDictionary<ISymbol, PreviewFeatureUsageType> requiresPreviewFeaturesSymbolsToUsageType = new();
 
-                if (context.Compilation.TryGetOrCreateTypeByMetadataName(WellKnownTypeNames.SystemRuntimeCompilerServicesRuntimeFeatureVirtualStaticsInInterfaces, out INamedTypeSymbol? runtimeFeatureTypeVirtualStaticsInInterfaces))
+                IFieldSymbol? virtualStaticsInInterfaces = null;
+                if (context.Compilation.TryGetOrCreateTypeByMetadataName(WellKnownTypeNames.SystemRuntimeCompilerServicesRuntimeFeature, out INamedTypeSymbol? runtimeFeatureType))
                 {
-                    ProcessPreviewAttribute(runtimeFeatureTypeVirtualStaticsInInterfaces, requiresPreviewFeaturesSymbols);
+                    virtualStaticsInInterfaces = runtimeFeatureType
+                        .GetMembers("VirtualStaticsInInterfaces")
+                        .OfType<IFieldSymbol>()
+                        .FirstOrDefault();
+                    if (virtualStaticsInInterfaces is null)
+                    {
+                        return;
```

Comment:
Is `virtualStaticsInInterfaces` required for analysis?

Reply:
I think technically it is ok to return here if `virtualStaticsInInterfaces ` is not found, because `RequiresPreviewFeatures` and `virtualStaticsInInterfaces ` are both new for .NET 6. However you are right that we should continue analysis even if it was null. Fixed now.